src/Sticky.js

```diff
@@ -8,15 +8,17 @@ export default class Sticky extends Component {
     topOffset: PropTypes.number,
     bottomOffset: PropTypes.number,
     relative: PropTypes.bool,
-    children: PropTypes.func.isRequired
+    children: PropTypes.func.isRequired,
+    isFooter: PropType.bool
```

Comment:
First, it's PropTypes, not PropType.

Second, I think `isFooter` is a bit implementation specific -- I'd rather see a prop called `edge` that accepts a couple of exported constants, (e.g. `export Edges = { TOP: 0, BOTTOM: 1 }`, and defaults to `Edges.TOP`